src/shared/Button.tsx

```diff
@@ -7,10 +7,10 @@ const schemaNameToClass: Record<ButtonSchema, string> = {
   secondary: "btn-secondary",
 };
 
-const Button: Component<{ children: JSX.Element; schema?: ButtonSchema }> = (
+const Button: Component<{ children: JSX.Element; schema?: ButtonSchema; class?:string }> = (
   props
 ) => (
-  <button class={schemaNameToClass[props.schema || "primary"]}>
+  <button class={schemaNameToClass[props.schema || "primary"] + (props.class ? ` ${props.class}` : '')}>
```

Comment:
I see that you added this so we can have margin/padding on the `Button` component themselves, right?

Ideally, I prefer having individual components only worry about their own content and placement, I think margin/padding values and other layout-related concerns should be left to the parent container. So for example instead of:

```jsx
<div class="flex">
  <Button class="mr-2">Button one</Button>
  <Button>Button two</Button>
</div>
```

I'd prefer doing:

```jsx
<div class="flex gap-2">
  <Button>Button one</Button>
  <Button>Button two</Button>
</div>
```

Could you refactor using that approach instead, so we can remove the `class` prop in the `Button` component?

Reply:
ok